CMakeLists.txt

```diff
@@ -87,7 +87,7 @@ set(GENERATED_DIRECTORY ${PROJECT_BINARY_DIR}/generated)
 file(MAKE_DIRECTORY ${GENERATED_DIRECTORY}/include)
 
 set(DOT_CONFIG_PATH ${GENERATED_DIRECTORY}/.config)
-set(CONFIG_H_PATH ${GENERATED_DIRECTORY}/include/config.h CACHE
+set(CONFIG_H_PATH ${GENERATED_DIRECTORY}/include/autoconfig.h CACHE
```

Comment:
I don't like autoconfig name as it may ppl think that it's autoconf output, also it's non-standard, better to have it like zephyr and kernel imho.
I understand that it's for extra safety, but it's easy to verify that all `config.h`s are removed and even if some1 includes it 2nd time it's not a problem at all.

Reply:
> I don't like autoconfig name as it may ppl think that it's autoconf output, also it's non-standard, better to have it like zephyr and kernel imho.

Nvm, I was wrong, there is `autoconf.h` in kernel, `config.h` is just wrapper, so feel free to make it `autoconf.h`.